### DISCUSSION TITLE
Wordmat 1.25 fejler på PC

### DISCUSSION THREAD

**Discussion by IBG-IT** [Eduap-com/WordMat#72] [Support] [answered]
Wordmat 1.25 på PC
 
PAS dog på! 

Jeg har fået Denne version 1.25 rullet den ud hurtigt, da mine matematiklærere gerne ville bruge den fra skoleårs start hos eleverne.
 
Desværre har det vist sig, at ca. 20% af lærernes maskiner med Word dør og ikke vil virke, når de har fået version 1.25!
Løsningen har været at fjerne 1.25 og lægge 1.24.2 på igen!
 
Jeg har ikke kunnet finde forskelle i Word versionerne og alle maskinernes Windows er også opdateret. Eneste lille forskel er at dem som dør måske har et versions nummer højere i Word end dem som Wordmat virker på!
 
Har andre oplevet fejl på Wordmat 1.25 på PC eller ide til hvad fejlen kan skyldes?

IT-koordinator på Ikast-Brande Gymnasium - Thue Godiksen

**Comment by Eduap-com** [comment]
Jeg har oplevet en enkelt der crashede ved første beregning, men det virkede efter en genstart.

**Reply by IBG-IT** [reply]
Det er ikke issuet desværre!  
Det er hele Word som dør og ikke bare Wordmat. Du kan overhovedet ikke bruge Word, når Wordmat 1.25 er installeret på de ramte maskiner .-(

**Reply by Eduap-com** [reply]
Hvad sker der præcist?
Sker der intet når man prøver at starte Word?
Kommer der en fejl?

**Reply by IBG-IT** [reply]
Word starter langsomt op og lukker så ned igen!  Uden Word helt har været startet...  Det er mystisk også da Word versionen ofte er "super" ny, hvor "almindelig ny" version ikke crasher... 

Nogle gange kommer Normal.dotm er ændret skal denne gemmes - men jeg er usikker pt. på om det er altid!

Jeg er ved at undersøge, som der kan være en rød tråd med at de maskiner som går ned har et bestemt tilføjelsesprogram installeret!  Da maskiner som har crashet har haft et eller andet installeret af tilføjelser...

**Reply by Eduap-com** [reply]
WordMat kører noget kode i opstarten. 
Hvis man i indstillinger har sat WordMat til at starte Maxima op, når Word starter, kan problemet være relateret til Maxima.
Det kunne forklare hvorfor det kun rammer nogle maskiner. En test på et par af mine maskiner gav dog ikke fejlen.

Kan jeg få dig til at checke om WordMat er sat til at starte op?
kør 'regedit'
find 'HKEY_CURRENT_USER\Software\WordMat\Settings'
Se om nøglen AutoStart er 0 eller 1

**Comment by IBG-IT** [comment]
Jeg er desværre ude af huset nu til fredag hvor jeg kigger på det!

Men har nu ses 3 fejl i dag med maxima kunne det være samme issue og evt løsning ?

Se også billede...


Mvh,
Thue Godiksen
________________________________
Fra: Eduap-com ***@***.***>
Sendt: tirsdag, august 16, 2022 21:28
Til: Eduap-com/WordMat ***@***.***>
Cc: Thue Godiksen ***@***.***>; Author ***@***.***>
Emne: Re: [Eduap-com/WordMat] Wordmat 1.25 fejler på PC (Discussion #72)


WordMat kører noget kode i opstarten.
Hvis man i indstillinger har sat WordMat til at starte Maxima op, når Word starter, kan problemet være relateret til Maxima.
Det kunne forklare hvorfor det kun rammer nogle maskiner. En test på et par af mine maskiner gav dog ikke fejlen.

Kan jeg få dig til at checke om WordMat er sat til at starte op?
kør 'regedit'
find 'HKEY_CURRENT_USER\Software\WordMat\Settings'
Se om nøglen AutoStart er 0 eller 1

**Reply by Eduap-com** [reply]
WordMat starter noget kode op når Word starter.
Maxima startes op hvis det er sat i indstillingerne. Det kan checkes her:
regedit
HKEY_CURRENT_USER\Software\WordMat\Settings
Check om AutoStart er sat til 0 eller 1

**Comment by IBG-IT** [comment]
Autostart står til 0
Fejlen og derefter fryser Wordmat – regner uendeligt…
”Der skete en fejl i forsøget på at starte Maxima – errCode: 2”

Jeg har nu set denne fejl på ca. 3 elev maskiner også – alle med version 1.25 og helt opdatet Word (f.eks. på denne Office jeg pt. Kigger på: version 2207 (Build 15427.20210)

Thue

Fra: Eduap-com ***@***.***>
Sendt: 16. august 2022 21:28
Til: Eduap-com/WordMat ***@***.***>
Cc: Thue Godiksen ***@***.***>; Author ***@***.***>
Emne: Re: [Eduap-com/WordMat] Wordmat 1.25 fejler på PC (Discussion #72)


WordMat kører noget kode i opstarten.
Hvis man i indstillinger har sat WordMat til at starte Maxima op, når Word starter, kan problemet være relateret til Maxima.
Det kunne forklare hvorfor det kun rammer nogle maskiner. En test på et par af mine maskiner gav dog ikke fejlen.

Kan jeg få dig til at checke om WordMat er sat til at starte op?
kør 'regedit'
find 'HKEY_CURRENT_USER\Software\WordMat\Settings'
Se om nøglen AutoStart er 0 eller 1

**Reply by IBG-IT** [reply]
Samme fejl er nu også begyndt at optræde, når du nedgradere til Wordmat 1.24.2 igen - så Wordmat stopper nu faktisk med at virke overhovedet :-(

**Reply by Eduap-com** [reply]
Er dette en anden fejl, end den du beskrev i starten. Det lyder som om den opstår ved beregning, således at Word kan startes op?

**Comment by MeatEasterPearls** [comment]
Jeg har oplevet samme problem med at Word crashede ved tryk på beregning (Wordmat 1.25 og Office 365 ajour).

Problemet blev løst ved at ændre "CAS motor" fra Maxima til GeoGebra i Wordmats indstillinger.

**Reply by Eduap-com** [reply]
Jeg er nu også stødt på en computer, hvor Word crashede ved første beregning. Fejlen opstår i den del, hvor der checkes for opdateringer.
Det kan altså umiddelbart løses ved at slå automatisk opdateringer fra under indstillinger | Avanceret

Hvorfor problemet kun opstår på nogle computere og hvorfor 1.24.2 virker, ved jeg ikke, men jeg tror godt det kan løses inden så længe.

**Comment by IBG-IT** [comment]
Korrekt nu starter Word men wordmat fryser i beregninger – så måske / måske ikke det samme som laver fejlen..  både 1.24.2 og 1.25…

Har fået et par løsningsforslag ind, men har ikke testet endnu!

ver. 1.25 til windows som ikke virker.
Afinstaller wordmat.
Online reparation af office.
Installer 1.24.2
Virker hver gang.

--
Gå ind under avanceret i indstillinger og genstart Wordmat.
https://www.youtube.com/watch?v=YtDc6c3fvx8


Fra: Eduap-com ***@***.***>
Sendt: 19. august 2022 13:16
Til: Eduap-com/WordMat ***@***.***>
Cc: Thue Godiksen ***@***.***>; Author ***@***.***>
Emne: Re: [Eduap-com/WordMat] Wordmat 1.25 fejler på PC (Discussion #72)


Er dette en anden fejl, end den du beskrev i starten. Det lyder som om den opstår ved beregning, således at Word kan startes op?

**Reply by MeatEasterPearls** [reply]
Har du prøvet at ændre "CAS motor" fra Maxima til GeoGebra i Wordmats indstillinger? Det fiksede problemet for mig (Wordmat 1.25).

**Comment by Eduap-com** [comment, marked as answer]
Jeg har nu lagt en ny version op til Windows 1.25.1 Der forhåbentlig løser problemet.

**Comment by IBG-IT** [comment]
Har desværre kun kunne få fingere i en maskine i dag... 
Version 1.25.1 løste ikke - at ligningsløsningen ikke hang, men ændring til Geogebra fra Maxima hjalp (som foreslået herinde)

**Reply by Eduap-com** [reply]
Der er to problemer omtalt herinde. 
1. At Word lukker helt ned ved ethvert forsøg på beregning eller endda ved opstart af Word.
2. Ved beregning fryser Wordmat – regner uendeligt… ”Der skete en fejl i forsøget på at starte Maxima – errCode: 2”

1.25.1 skulle gerne løse problem 1.

Problem 2 kan skyldes mange forskellige ting. Det opstår hvis WordMat ikke kan starte eller få forbindelse til Maxima. Det skyldes typisk antivirus, der blokerer.

At vælge GeoGebra som alternativ CAS-motor er netop indført for at omgå problemer hvor antivirus blokerer, men det er bedst at få Maxima til at køre, da der ting der kun virker med Maxima.

**Comment by IBG-IT** [comment]
Problem 1:At Word lukker helt ned ved ethvert forsøg på beregning eller endda ved opstart af Word.

Ser ud til at være løst med version 1.25.1 da word nu kører fejlfrit og Wordmat virker..
...

Problem 2: Ved beregning fryser Wordmat – regner uendeligt… ”Der skete en fejl i forsøget på at starte Maxima – errCode: 2”

Er løst og skyldes at alt gammelt software ikke blev slettet på PC ved ny Wordmat.
- Fjernede Wordmat og genstartede.
- Fjernede/slette mappen C: / programmer / Wordmat   (den havde rester af gl. Maxima filer / programmer)  og genstartede så PC
- Installerede Wordmat 1.25.1 og alt virkede .-)

**Reply by Eduap-com** [reply]
Mange tak for beskrivelsen af løsningen til problem 2. Det må kunne løses. Jeg undersøger det nærmere.

**Reply by Eduap-com** [reply]
Der er nu lagt en ny version 1.25.2 op, der rydder bedre op.
Hvis errcode 2 problemet opstår, løses det ved at installere 1.25.2 to gange, da der så bliver ryddet op ved afinstallation efter den første installation.
